Dockerfile

```diff
@@ -27,7 +27,7 @@ FROM ${TOOLCHAIN} AS toolchain
 RUN apk --update --no-cache add bash curl build-base protoc protobuf-dev
 
 # build tools
-FROM toolchain AS tools
+FROM --platform=${BUILDPLATFORM} toolchain AS tools
```

Comment:
I think this might not work correctly, but I might be wrong (whether platform should be set on `toolchain`).

Reply:
`BUILDPLATFORM` uses the local one where the build happens, the talos dockerfile also uses the same format. So it should it build fine locally

Reply:
yep, I know, I implemented this, but when I was doing it (long time ago), I had to push `--platform=${BUILDPLATFORM}` to the very first stage in the chain, otherwise it got ignored

Reply:
but if you verified that it works, I have nothing against it ;) just saying my previous experience with it

Reply:
> yep, I know, I implemented this, but when I was doing it (long time ago), I had to push --platform=${BUILDPLATFORM} to the very first stage in the chain, otherwise it got ignored

this is still on the first one in the chain https://github.com/frezbo/kres/blob/chore/go-tools-multiarch-crossbuild/Dockerfile#L26

`tools` and `base` inherit `toolchain`